fractal_server/migrations/versions/84bf0fffde30_add_dumps_to_applyworkflow.py

```diff
@@ -0,0 +1,91 @@
+"""Add dumps to ApplyWorkflow
+
+Revision ID: 84bf0fffde30
+Revises: 99ea79d9e5d2
+Create Date: 2023-10-26 16:11:44.061971
+
+"""
+import sqlalchemy as sa
+from alembic import op
+
+# revision identifiers, used by Alembic.
+revision = "84bf0fffde30"
+down_revision = "99ea79d9e5d2"
+branch_labels = None
+depends_on = None
+
+
+def upgrade() -> None:
+    # ### commands auto generated by Alembic - please adjust! ###
+    with op.batch_alter_table("applyworkflow", schema=None) as batch_op:
+        batch_op.add_column(
+            sa.Column(
+                "user_dump",
+                sa.String(),
+                server_default="__UNDEFINED__",
+                nullable=False,
+            )
+        )
+        batch_op.add_column(
+            sa.Column(
+                "input_dataset_dump",
+                sa.JSON(),
+                server_default="{}",
+                nullable=False,
+            )
+        )
+        batch_op.add_column(
+            sa.Column(
+                "output_dataset_dump",
+                sa.JSON(),
+                server_default="{}",
+                nullable=False,
+            )
+        )
+        batch_op.alter_column(
+            "workflow_dump", existing_type=sa.JSON(), nullable=False
```

Comment:
Missing test:
* Start (before this migration) with a row with `ApplyWorkflow.workflow_dump=None`
* Apply the migration

Reply:
If this fails, let's add a `server_default` also here (if it exists in `alter_column`)